test_scripts/API/CancelInteraction/006_SendCancelInteraction_WrongCancelID_HMIRespondsIgnored.lua

```diff
@@ -53,16 +61,19 @@ local function SendCancelInteraction()
   local hmiSession = common.getHMIConnection()
   
   local cid0 = mobileSession:SendRPC(rpcInteraction.name, rpcInteraction.params)
-  local cid1 = mobileSession:SendRPC(rpcCancelInteraction.name, rpcCancelInteraction.params)
+  local interaction_id = 0
   
-  EXPECT_HMICALL(rpcInteraction.hmi_name, {})
+  EXPECT_HMICALL(rpcInteraction.hmi_name, rpcInteraction.hmi_params)
   :Do(function(_, data)
-    hmiSession:SendResponse(data.id, data.method, "SUCCESS", {})
+    interaction_id = data.id
   end)
 
+  local cid1 = mobileSession:SendRPC(rpcCancelInteraction.name, rpcCancelInteraction.params)
```

Comment:
This should probably be in the `Do` block for 
```
EXPECT_HMICALL(rpcInteraction.hmi_name, rpcInteraction.hmi_params)
```
at least.

Reply:
moved in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/085ecf3126b192f8562f75bf9b3718c6cdde3111

Reply:
i removed this. putting this line in the callback creates a race condition. I could use wait() but it isn't any better of a test so I'd rather not.